content/en/getting_started/tagging/unified_service_tagging.md

```diff
@@ -405,6 +410,39 @@ instances:
 ### Serverless environment
 
 For more information about AWS Lambda functions, see [how to connect your Lambda telemetry using tags][15].
+
+### OpenTelemetry environment
+
+In OpenTelemetry environments, you can use the [OpenTelemetry Collector][16] to collect and batch telemetry data (like metrics, traces, and logs) and the [OpenTelemetry Exporter][17] to send the data to Datadog (without the Datadog Agent).
```

Comment:
We also support using the Datadog Agent as an OpenTelemetry setup https://docs.datadoghq.com/opentelemetry/ we should clarify what to do in that situation